el8/haproxy/haproxy.cfg

```diff
@@ -0,0 +1,64 @@
+global
+    log 127.0.0.1 local0 notice
+    maxconn 10000
+    user haproxy
+    group haproxy
+    stats socket /var/lib/haproxy/haproxy.sock mode 600 level admin
+    stats timeout 10m
+
+    ssl-default-bind-options prefer-client-ciphers
+    ssl-default-bind-ciphersuites TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256
+    ssl-default-bind-ciphers ECDH+AESGCM:ECDH+CHACHA20:ECDH+AES256:ECDH+AES128:!aNULL:!SHA1:!AESCCM
+    ssl-default-server-ciphersuites TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256
+    ssl-default-server-ciphers ECDH+AESGCM:ECDH+CHACHA20:ECDH+AES256:ECDH+AES128:!aNULL:!SHA1:!AESCCM
+
+defaults
+    mode http
+    retries 3
+    option redispatch
+    timeout connect 5s
+    timeout client 30s
+    timeout server 10s
+    option httpchk GET /
```

Comment:
Shouldn't this be also included in el9?

Reply:
this missing default makes `haproxy -c -V -f /etc/kazoo/haproxy/haproxy.cfg` complain in EL9 for bigcouch

Reply:
can you post the error? I didn't see one in rocky9

Reply:
it is a warning @crecinos420 any chance you still have a capture or can reproduce to get the warnings we had about this?

Reply:
I didn't see the warnings when starting up but I did apply the same configs i did for el8 there and they seem to be working fine now, good catch!